sdk/identity/identity/src/credentials/defaultAzureCredential.ts

```diff
@@ -65,7 +65,7 @@ export class UnavailableDefaultCredential implements TokenCredential {
  * - {@link AzureCliCredential}
  * - {@link AzurePowerShellCredential}
  * - {@link AzureDeveloperCliCredential}
- * - {@link BrokerCredential}
+ * - BrokerCredential (a broker-enabled credential that requires @azure/identity-broker is installed)
```

Comment:
This should fix the lint error
```suggestion
 * - BrokerCredential (a broker-enabled credential that requires \@azure/identity-broker is installed)
```